js/index.js

```diff
@@ -134,3 +134,45 @@ messageForm.addEventListener("submit", function (event) {
   // Inside the callback function, add a new line of code to clear the form
   messageForm.reset();
 });
+
+// Fetch GitHub Repositories
+const githubRequest = new XMLHttpRequest();
+
+githubRequest.open("GET", "https://api.github.com/users/DoinaFitchevici/repos");
+githubRequest.send();
+
+// Handle Response from Server
+
+githubRequest.addEventListener("load", (event) => {
```

Comment:
There's a risk of a race condition here: It's unlikely, but imagine if the response came back before the event handler were defined.